packages/node_modules/@node-red/editor-client/src/js/ui/palette-editor.js

```diff
@@ -1462,6 +1470,63 @@ RED.palette.editor = (function() {
         })
     }
 
+    const updateStatusWidget = $('<button type="button" class="red-ui-footer-button red-ui-update-status"></button>');
+    let updateAvailable = [];
+
+    function addUpdateInfoToStatusBar() {
+        updateStatusWidget.on("click", function (evt) {
+            RED.actions.invoke("core:manage-palette", {
+                view: "nodes",
+                filter: '"' + updateAvailable.join('", "') + '"'
+            });
+        });
+
+        RED.popover.tooltip(updateStatusWidget, function () {
+            const count = updateAvailable.length || 0;
+            return RED._("palette.editor.updateCount", { count: count });
+        });
+
+        RED.statusBar.add({
+            id: "update",
+            align: "right",
+            element: updateStatusWidget
+        });
+
+        updateStatus({ count: 0 });
+    }
+
+    function refreshUpdateStatus() {
+        updateAvailable = [];
+
+        for (const module of Object.keys(nodeEntries)) {
+            if (loadedIndex.hasOwnProperty(module)) {
+                const moduleInfo = nodeEntries[module].info;
+                if (moduleInfo.pending_version) {
+                    // Module updated
+                    continue;
+                }
+                if (updateAllowed &&
+                    semVerCompare(loadedIndex[module].version, moduleInfo.version) > 0 &&
+                    RED.utils.checkModuleAllowed(module, null, updateAllowList, updateDenyList)
+                ) {
+                    updateAvailable.push(module);
+                }
+            }
+        }
+
+        updateStatus({ count: updateAvailable.length });
```

Comment:
Have not debugged yet, but I'm only seeing a count of `1` in the widget, but when I open the palette manager, there are over 10 marked as having updates available.

Reply:
Can you try to add a node to the workspace and see if the count has changed.

Reply:
In the `init` I did not put `refreshUpdateStatus()` to all the events, maybe that's it.

Reply:
Can you check if adding `refreshUpdateStatus()` to `'registry:node-type-added'` fixes the issue.

Reply:
Looks good. My only other comment I was about to add before you pushed the commit was the call to `refreshUpdateStatus()` should be moved into the `finally` block - line 473. If you have multiple catalogues to load (as I do locally for example), no point refreshing the widget until all catalogues have been loaded.